spark/dl/src/test/scala/com/intel/analytics/bigdl/torch/TorchSpec.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.torch
+import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
+
+class TorchSpec extends FlatSpec with BeforeAndAfter with Matchers {
+  def torchCheck(): Unit = {
```

Comment:
I switch to use `beforeAll` but it causes `mvn clean package` to fail. I think `beforeAll` is like the `before`s we currently have for each test .

```
*** 120 SUITES ABORTED ***
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] BigDL .............................................. SUCCESS [  1.064 s]
[INFO] spark_bigdl ........................................ SUCCESS [  0.010 s]
[INFO] spark-version ...................................... SUCCESS [  0.010 s]
[INFO] 1.5-plus ........................................... SUCCESS [  9.350 s]
[INFO] bigdl .............................................. FAILURE [17:16 min]
[INFO] dist-spark-1.5.1-scala-2.10.5-linux64 .............. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 17:27 min
[INFO] Finished at: 2017-05-01T12:23:58+08:00
[INFO] Final Memory: 131M/6454M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.scalatest:scalatest-maven-plugin:1.0:test (test) on project bigdl: There are test failures -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :bigdl

```